docs/topics/mta-rn-new-features-7-0-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-7-0-0_{context}"]
+= New features
+
```

Comment:
The new feature list is missing : 
1) Enhanced assessment module with custom questionnaire
2) Grouping applications for assessment into archetypes.
3) Unlink applications from JIRA

Reply:
>Unlink applications from JIRA
Isn't really something I would call out, but I would add.
3. Dynamic querying and aggregation of analysis data